lib/index.js

```diff
@@ -51,30 +52,45 @@ cliCompleter.pluginId = constants.sourceId
  * Plugin to validate that Markdown links and images reference existing local
  * files and headings.
  *
- * @type {import('unified').Plugin<[Options?, FileSet?]|[Options?]|void[], Root>}
+ * @typedef {import('unified-lint-rule').Label|import('unified-lint-rule').Severity} Severity
+ * @type {import('unified').Plugin<[(Options|[Severity, Options?])?, FileSet?], Root>}
+ * https://github.com/microsoft/TypeScript/pull/48132
+ * @param [rawOptions]
+ * @param [fileSet]
```

Comment:
- The `typedef` should not be in this block: that makes the description belong to *it*. The description belongs to the plugin. You can move the typedef up.
- Severity and label are different things as types, severity does not encompass both, so I think it’s better to not import *both* as one name, but import them separately under their own names
- https://github.com/remarkjs/remark-lint/pull/283#discussion_r820542311